### DISCUSSION TITLE
Share account between different users

### DISCUSSION THREAD

**Discussion by FDumarey** [firefly-iii/firefly-iii#8150] [Firefly III]
Hello, looks like a very interesting PFM.
At home I have 2 users: me and my wife. We can both do our personal accounts using our account.
What is the best practice to manage shared accounts? I read somewhere you cannot share accounts between users.

**Comment by JC5** [comment]
Right now, unfortunately, you'll have to share passwords and use 1 shared user-account.

**Reply by tukusejssirs** [reply]
This is a pity. I believe many of us share finances with their partner, many of us have a family, or even projects / (small) companies would benefit from this. Sharing credentials / using a single user account is not a good idea.

I wish we could share any and all assets, budgets, expenses, piggy banks (OFC, not by default), or even view any and all transactions.

**Comment by FDumarey** [comment]
Okay, no problem!
Thanks for the quick response.

Would be a nice feature in the future, but I understand then you would 
have problems differntiating personal and shared accounts.


------ Oorspronkelijke bericht ------
Van "James Cole" ***@***.***>
Aan "firefly-iii/firefly-iii" ***@***.***>
CC "Frederik Dumarey" ***@***.***>; "Author" 
***@***.***>
Datum 14/11/2023 15:17:56
Onderwerp Re: [firefly-iii/firefly-iii] Share account between different 
users (Discussion #8150)
